CONTRIBUTING.rst

```diff
@@ -0,0 +1,76 @@
+- Prepare a commit for a release candidate, get an approving review, and tag the commit which will upload artifacts to PyPI.
+- Notify people and give time for feedback.
+- Address concerns raised via new issues for later handling or PRs against the release branch.
+- Prepare a commit for a final release, get an approving review, and tag the commit which will upload artifacts to PyPI.
+
+
+Step-by-step
+------------
+
+.. note::
+
+    Commands are written with Linux in mind and a ``venv`` located in ``venv/``.
+    Adjust per your OS and virtual environment location.
+    For example, on Windows with an environment in the directory ``myenv/`` the Python command would be ``myenv/scripts/python``.
+
+- Define the final release version you are preparing.
+
+  - towncrier uses `CalVer <https://calver.org/>`_ of the form ``YY.MM.micro`` with the micro version just incrementing.
+  - Normalize the version according to `PEP 440 <https://www.python.org/dev/peps/pep-0440/#normalization>`_.
+
+    - This requires that ``towncrier[dev]`` extra is installed.
+    - ``venv/bin/python admin/canonicalize_version.py 19.09.00-rc1``
+    - Outputs ``19.9rc1`` which is the form to be used.
+
+- Create a release branch with a name of the form ``release-19.9`` starting from the ``master`` branch.
+
+  - This new branch will contain all tagged release candidate commits as well as the final tagged release commit.
+
+- Update the version to the release candidate with the first being ``rc1`` (as opposed to 0).
+
+  - In ``src/towncrier/_version.py`` the version is set using ``incremental`` such as ``__version__ = Version('towncrier', 19, 9, release_candidate=1)``
+
+- Run ``venv/bin/towncrier build --yes`` to build the the newsfragments into the release notes document and remove the newsfragment files.
+
+- Commit and push to the primary repository, not a fork.
+
+  - It is important to not use a fork so that pushed tags end up in the primary repository.
+
+- If working on the first release candidate from this branch, create a PR named in the form ``Release 19.9``.
+
+- Request a review and address raised concerns until receiving an approval.
+
+- Tag that commit such as ``19.9rc1`` and push the tag to the primary repository.
+
+  - This will result in another build which will publish to PyPI.
+  - Confirm the presence of the release on PyPI.
+  - # TODO: use the GitHub release and tagging feature?
+
+- If not processing the final release, `dismiss the approving review <https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/dismissing-a-pull-request-review>`_.
+
+  - The review process will be reused for any subsequent release candidates as well as the final release so it must be cleared at each stage.
+
+- # TODO: notify people some how some way, the maillist?  hmmm...
+
+- If another release candidate is required:
+
+  - Submit PRs against the release branch to integrate the needed changes.
```

Comment:
my idea is to fix master first...and then the release branch... but this is also ok.

Reply:
I guess as long as the cherry pick from master is done as a PR against the release branch it doesn't really matter.  My point is to encourage that the actual release branch stays reasonably clean to alleviate confused tossing of changes there and flailing around trying to fix something.

```suggestion
  - Submit PRs against the release branch to integrate the needed changes.  Any PRs could be cherry picks from the ``master`` branch if already resolved there, or direct PRs against the release branch that will be merged back into master at the completion of the release.
```